bitnami/spark/templates/statefulset-master.yaml

```diff
@@ -71,28 +71,27 @@ spec:
             - name: SPARK_MASTER_OPTS
               value: {{ .Values.master.configOptions | quote }}
             {{- end }}
-            {{- if eq ( .Values.security.rpc.authenticationEnabled | quote ) "yes" }}
+            {{- if .Values.security.rpc.authenticationEnabled }}
```

Comment:
The spark container expects `yes/no` for boolean variables. Changing it here would also need the spark container to work with `true/false` as values for boolean variables. Could you please describe the issue this PR attempts to fix? Please share the issue number if there is an associated issue. Or possible share details how to reproduce the issue this PR attempts to fix

Reply:
The container works with true/false. The use of "yes" and "no" is bad practice in Helm, this is a boolean value and should be true/false. Other charts in bitnami uses true/false.

Reply:
Also had issue with the version of Helm I was using. If I recall correctly it was interpretting yes/no as booleans. Which did not work with the string compare test done in the chart. The values "yes" and "no" needed to be quoted or else they would be interpreted as boolean and were not processed properly by the chart. It's just a meassy way to specify boolean values which should be true/false.

Reply:
@cccs-jc thanks for the information.